test/nextjournal/clerk/viewer_test.clj

```diff
@@ -338,6 +338,14 @@
       (is (every? (every-pred not-empty string?) ids))
       (is (distinct? ids))))
 
+  (testing "Fragments renders values from def vars"
+    (is (= 3
+           (-> (eval-test/eval+extract-doc-blocks "(ns nextjournal.clerk.viewer-test.fragments (:require [nextjournal.clerk :as clerk]))
+(clerk/fragment
+ 1 2 (def x 3))")
+               last :nextjournal/value :nextjournal/presented :nextjournal/value))))
```

Comment:
A test for `clerk/comment` would be great as well. Maybe also nesting `clerk/comment`s.